prow/config.yaml

```diff
@@ -479,6 +479,14 @@ presets:
       secretKeyRef:
         name: authentikos-token
         key: token
+- labels:
+    preset-override-deps: "release-1.4"
+  env:
+  - name: DEPENDENCIES
+    valueFrom:
+      configMapKeyRef:
+        name: release-1.4-deps
+        key: dependencies
```

Comment:
Do we have this configmap defined somewhere?

Reply:
Yes, it is  in the private cluster `test-pods` namespace.